index.html

```diff
@@ -34,16 +27,14 @@ <h1 class="header">I’m Ngala Mac
                 <div class="links">
                     <h3><a href="#">LET’S CONNECT</a></h3>
                     <ul>
-                        <li><img src="./Images/twitter.png"></li>
-                        <li><img src="./Images/Linkedin icon.png"></li>
-                        <li><img src="./Images/medium.png"></li>
-                        <li><img src="./Images/github.png"></li>
-                        <li><img src="./Images/twitter.png"></li>
+                        <li><a href="#"><img src="./Images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./Images/Linkedin icon.png"></a></li>
+                        <li><a href="#"><img src="./Images/medium.png"></a></li>
+                        <li><a href="#"><img src="./Images/github.png"></a></li>
                      </ul>
```

Comment:
- [ ] Kindly it will be better 😃 if you increase little the margin between the text```LET’S CONNECT ``` and the social icons you here in your design the margin between them is less than the Figma Design, so let's 💯 increase the margin between them a little to stick to the Figma design and make the design nice for the users 🚀😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/fe4d22a5-dbd9-40ae-9a0d-e9ca888fb329)   |    ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/efc52a4f-9cfe-485b-a90d-35d127826b0b) |